pyccel/codegen/printing/fcode.py

```diff
@@ -2657,6 +2657,30 @@ def _print_NumpyUfuncBase(self, expr):
         code = '{0}({1})'.format(func_name, code_args)
         return self._get_statement(code)
 
+    def _print_NumpySign(self, expr):
+        """ Print the corresponding Fortran function for a call to Numpy.sign
+
+        Parameters
+        ----------
+            expr : Pyccel ast node
+                Python expression with Numpy.sign call
+
+        Returns
+        -------
+            string
+                Equivalent internal function in Fortran
+
+        Example
+        -------
+            import numpy
+
+            numpy.sign(x) => numpy_sign(x)
+            numpy_sign is an interface which calls the proper function depending on the data type of x
+
+        """
+        self._additional_imports.add(Import('numpy_f90', Module('numpy_f90',(),())))
```

Comment:
Can you import the `numpy_sign` function instead of all the module? I don't remember the syntax off the top of my head, but it should be something like:
```
self._additional_imports.add(Import('numpy_f90', Module('numpy_f90',('numpy_sign',),())))
```

Reply:
I tried to put it in `funcs` like this:
self._additional_imports.add(Import('numpy_f90', Module('numpy_f90',(),('numpy_sign',)))) but it doesn't work.

Reply:
There is an example here: https://github.com/pyccel/pyccel/blob/master/pyccel/codegen/printing/fcode.py#L372

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/11d147f49d0a936d3d6167cd71b4b6492dc2c903

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/869205ede015568fca2bc1e9f3fc30591954f916